mighty/mighty_exploration/mighty_exploration_policy.py

```diff
@@ -111,8 +116,10 @@ def sample_func_logits(self, state_array):
         # ─── Continuous squashed‐Gaussian (4‐tuple) ──────────────────────────
         elif isinstance(out, tuple) and len(out) == 4:
             action = out[0]  # [batch, action_dim]
+            
+            print(f'Self Model : {self.model}')
             log_prob = sample_nondeterministic_logprobs(
-                z=out[1], mean=out[2], log_std=out[3], sac=self.algo == "sac"
+                z=out[1], mean=out[2], log_std=out[3], sac=isinstance(self.model, SACModel)
```

Comment:
Bad idea! What If I want to implement a different model class for SAC that e.g. handles prediction differently? Then the policy stops functioning.